### PR TITLE
Post Snappy messages to Jembi

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr ready for review

File: snappybouncer/tasks.py [hunk 1]
```diff
@@ -24,6 +26,50 @@ def send_helpdesk_response(ticket):
...
+def jembi_format_date(date):
+    return date.strftime("%Y%m%d%H%M%S")
+
+
+def extract_class_from_tags(tags):
+    # ["@person", "#coffee", "#payment"] -> "coffee"
+    for tag in tags:
+        if tag[0] == "#":
+            return tag[1::]
+
```
Comment (gsvr) [inline snappybouncer/tasks.py:48]: I think we should return something here if the ticket wasn't tagged right - maybe default to "other"?

File: snappybouncer/tasks.py [hunk 2]
```diff
@@ -24,6 +26,50 @@ def send_helpdesk_response(ticket):
...
+def build_jembi_helpdesk_json(ticket, tags, operator_num):
+    json_template = {
+        "encdate": jembi_format_date(ticket.created_at),
+        "repdate": jembi_format_date(ticket.updated_at),
+        "mha": 1,
+        "swt": 2,  # 1 ussd, 2 sms
+        "cmsisdn": ticket.msisdn,
+        "dmsisdn": ticket.msisdn,
+        "faccode": None,  # TODO look this up in vumi when available
```
Comment (gsvr) [inline snappybouncer/tasks.py]: Lookup required.

**Comment by gsvr** [comment]
@imsickofmaps I pushed some fixes, have a look if you're happy with them.

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**